helper/random-string.js

```diff
@@ -0,0 +1,19 @@
+// Generate random strings.
+const generateRandom = (length = 32, alphanumeric = true) => {
```

Comment:
no need to create new helper for it.
add it into common-function helper

Reply:
made the required changes